### DISCUSSION TITLE
add to list small own project which using Jolt

### DISCUSSION THREAD

**Discussion by wagnerks** [jrouwe/JoltPhysics#883] [Q&A] [answered]
Hi!
First of all, I want to say that Jolt is a great project. I really like how it is written and how it works.

I wonder if it's okay to add my [project](https://github.com/wagnerks/KillRealGameEngine) to the list of Jolt-using projects. 
It is still relatively small, and I am not sure whether every project that uses Jolt should be added there, or if it is reserved for larger, released, or more serious projects.

If the list is open to any projects, I would open a PR. If not, I'll come back when my project becomes more fitting :)

**Comment by jrouwe** [comment, marked as answer]
Hello, I've looked at your project and at the moment I think it is a bit too small to be mentioned. Also, it only implements the bare minimum of Jolt, so is not really a showcase. Ask again if it gains some popularity or if you make something really cool with it.
